package.json

```diff
@@ -87,7 +87,11 @@
     "react-html-parser": "^2.0.2",
     "react-player": "^2.1.1",
     "react-responsive": "^8.1.0",
-    "simplebar-react": "^2.2.0"
+    "simplebar-react": "^2.2.0",
```

Comment:
Did we change/add that much of features using new npms?

Reply:
This package is used for *pretty* scrollbar in **About link**
But it caused some problems on public pages as we have the linking problem and there is no scroll on public pages for about link. 
If we cannot fix it after linking problem is gone then I will remove it.

Reply:
Okay. Got it.